vivisect/parsers/pe.py

```diff
@@ -286,7 +287,7 @@ def loadPeIntoWorkspace(vw, pe, filename=None):
                 vw.markDeadData(secbase, secbase+len(secbytes))
 
         except Exception, e:
-            print "Error Loading Section (%s size:%d rva:%.8x offset: %d): %s" % (secname,secfsize,secrva,secoff,e)
+            logging.getLogger("parsers.pe.loadPE").warning("Error Loading Section (%s size:%d rva:%.8x offset: %d): %s", secname, secfsize, secrva, secoff, e)
```

Comment:
Why not add something at the top like `logger = logging.getLogger(__name__)`? Grabbing the logger on every exception is unnecessary. It would end up being very similar with the name being `vivisect.parsers.pe` (imo function names should not be in a logger name) .

I would update this in `vivisect/__init__.py` too.